addons/udes_security/common.py

```diff
@@ -0,0 +1,14 @@
+from odoo.http import request
+import re
+
+
+# Replaces redirect param with None if it's going external
+def remove_external_redirect(redirect):
+    base_location = request.httprequest.url_root.rstrip('/')
```

Comment:
Python convention is that the docstring for a method/class should be in triple double-quotes, immediately below the def:
```
def remove_external_redirect(redirect):
    """Replaces redirect param with None if it's going external"""
```